tests/handler/utils/config.py

```diff
@@ -0,0 +1,17 @@
+from cytoolz import (
+    merge,
+)
+
+from contracts.utils.config import (
+    get_sharding_config,
+)
+
+
+def get_sharding_testing_config():
+    REPLACED_PARAMETERS = {
+        'NOTARY_LOCKUP_LENGTH': 120,
+    }
```

Comment:
if for now we just have one key to be replaced, seems we can use `cytoolz.dicttoolz.assoc`.(http://toolz.readthedocs.io/en/latest/api.html#toolz.dicttoolz.assoc)